MaterialSkin/HTML/material/html/js/browse-page.js

```diff
@@ -1595,17 +1597,53 @@ var lmsBrowse = Vue.component("lms-browse", {
         },
         setScrollElement() {
             this.scrollElement = document.getElementById(this.useGrid ? "browse-grid" : "browse-list");
+            var isScrolling;
             this.scrollElement.addEventListener('scroll', () => {
-                if (this.fetchingItems || this.listSize<=this.items.length) {
-                    return;
-                }
+                // display a letter as overlay whilst scrolling - displays first letter of item at top of screen
+
+                window.clearTimeout(isScrolling);                            // Clear a timeout if still scrolling
+                isScrolling =  setTimeout( () => {                           // Show the overlay for 0.5 sec after scrolling...
+                               this.showLetter = false;
+                               },500);
+
                 const scrollY = this.scrollElement.scrollTop;
                 const visible = this.scrollElement.clientHeight;
                 const pageHeight = this.scrollElement.scrollHeight;
                 const pad = (visible*2.5);
 
                 const bottomOfPage = (visible + scrollY) >= (pageHeight-(pageHeight>pad ? pad : 300));
 
+                const numPages =  Math.floor(scrollY/visible)+1;            // Calculate number of pages scrolled
+                var stopWordsRE = /(?:^|\s+)(?:the|The|El)(?=\s+|$)/gi;     // Pathetic attempt at stop word removal (needs to be a parameter)
```

Comment:
I _think_ LMS might be able to list the sort-character, in which case this RE would not be required.

Reply:
Yes I think it can, couldn't see it how do it here so bodged a simple test.

Reply:
Yeah, the request for artists/albums/genres would need to be updated to ask for their 'textkey' However, not sure how this would work for apps (e.g. Spotify)